api/v1/logic.go

```diff
@@ -90,12 +97,15 @@ func renderRuntimeTransaction(storageTransaction client.RuntimeTransaction) (api
 					return fmt.Errorf("to: %w", err2)
 				}
 				to := string(toAddr)
+				toEth := ethCommon.BytesToAddress([]byte(storageTransaction.AddressLookup[body.To.String()])).String()
```

Comment:
here and elsewhere: Can you use [ethCommon.HexToAddress](https://pkg.go.dev/github.com/ethereum/go-ethereum/common#HexToAddress) instead of the byte-wrangling?

Reply:
Yeah, this is better.